draft-ietf-acme-acme.md

```diff
@@ -306,36 +306,25 @@ JWS objects sent in ACME requests MUST meet the following additional criteria:
 * The JWS MUST NOT have the value "none" in its "alg" field
 * The JWS Protected Header MUST include the following fields:
   * "alg"
-  * "jwk"
+  * "kid" (except for new-reg and revoke-cert requests).
   * "nonce" (defined below)
   * "url" (defined below)
 
-Note that this implies that GET requests are not authenticated.  Servers MUST
+The "kid" field should contain a registration URL received by POSTing to the
+new-reg resource. The "kid" field is not required when POSTing to the new-reg
+resource, because no such URL is available to the client yet. The "kid" field
+may be omitted for revoke-cert requests if they are signed by the
+certificate key rather than an account key.
```

Comment:
This para should lay out the rules explicitly:
* For new-reg requests, there MUST be a `"jwk"` field containing the key being registered.
* For revoke-cert requests, there MUST be a`"jwk"` field containing a key authorized to revoke the cert.
* For all other requests, there MUST be a `"kid"` field containing the account URI